packages/xo-server-backup-reports/src/index.js

```diff
@@ -145,73 +220,176 @@ class BackupReportsXoPlugin {
     this._xo.on('job:terminated', this._report)
   }
 
-  test({ runId }) {
-    return this._backupNgListener(undefined, undefined, undefined, runId)
+  async test({ runId }) {
+    return this._report(runId)
   }
 
   unload() {
     this._xo.removeListener('job:terminated', this._report)
   }
 
-  _wrapper(status, job, schedule, runJobId) {
-    if (job.type === 'metadataBackup') {
-      return
-    }
+  async _wrapper(runJobId, { type, status, force } = {}) {
```

Comment:
should be named `_report`, it's weird that the bound version of `_wrapper` is called differently.